### ISSUE TITLE
Problem when open dataset with labelme

### ISSUE THREAD

**Issue by jisheng047** [wkentaro/labelme#307]
** (python3:16383): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-wC2O370H2k: Connection refused
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/labelme/app.py", line 854, in fileSelectionChanged
    self.loadFile(filename)
  File "/usr/local/lib/python3.5/dist-packages/labelme/app.py", line 1137, in loadFile
    self.imageData = self.loadImageFile(filename)
  File "/usr/local/lib/python3.5/dist-packages/labelme/app.py", line 1182, in loadImageFile
    image_pil = apply_exif_orientation(image_pil)
  File "/usr/local/lib/python3.5/dist-packages/labelme/app.py", line 1576, in apply_exif_orientation
    orientation = exif['Orientation']
KeyError: 'Orientation'
[1]    16383 abort (core dumped)  labelme

Please help me solve this error

**Comment by wkentaro** [comment]
Please run below to use fixed version of labelme
```
pip install labelme==3.7.2 --no-cache-dir
```

**Comment by mfaramarzi** [comment]
> pip install labelme==3.7.2 --no-cache-dir

I tried this version as well and still getting: **Aborted (core dumped)**
